core/common/src/main/java/alluxio/RuntimeConstants.java

```diff
@@ -33,8 +33,11 @@
   public static final String ALLUXIO_JAR =
       "target/alluxio-" + VERSION + "-jar-with-dependencies.jar";
 
-  /** The URL to Alluxio documentation for this version on project web site. */
+  /** The URL of Alluxio documentation for this version on project web site. */
   public static final String ALLUXIO_DOCS_URL;
 
+  /** The URL of Alluxio debugging documentation. */
+  public static final String ALLUXIO_DEBUG_DOCS_URL = ALLUXIO_DOCS_URL + "/en/Debugging-Guide.html";
```

Comment:
What is the ordering guarantee of the static blocks? The block above (that sets `ALLUXIO_DOCS_URL`) will always execute before this line?